draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -217,13 +218,49 @@ Each member zone MAY have one or more additional properties, described in this c
 These properties are completely optional and the catalog zone consumer SHOULD ignore those it does not understand.
 Properties are represented by RRsets below the corresponding member node.
 
+## The Change of ownership (Coo) Property {#cooproperty}
+
+The 'coo' property facilitates controlled migration of a member zone from one catalog to another.
+
+A Change Of Ownership is signaled by the 'coo' property in the catalog zone currently ``owning'' the zone.
+The name of the new catalog is in the value of a PTR record in the old catalog.
+For example if member "example.com." will migrate from catalog zone `$OLDCATZ` to catalog zone `$NEWCATZ`, this appears in the `$OLDCATZ` catalog zone as follows:
+
+```
+<unique-N>.zones.$OLDCATZ 0 IN PTR example.com.
+coo.<unique-N>.zones.$OLDCATZ 0 IN PTR zones.$NEWCATZ
+```
+
+The PTR RRset MUST consist of a single PTR record.
+A catalog zone consumer MUST not process and ignore PTR RRsets with more than a single record.
+
+When a catalog zone consumer of catalog zone `$OLDCATZ` receives an update which adds or changes a `coo` property for a member zone in `$OLDCATZ` signalling a new owner `$NEWCATZ`, it does *not* migrate the member zone immediately.
+This is because the catalog zone consumer may not have the `<unique-N>` identifier associated with the member zone in `$NEWCATZ` and because name servers do not index Resource Records by RDATA, it does not know wether or not the member zone is configured in `$NEWCATZ` at all.
+It has to wait for an update of `$NEWCATZ` adding or changing that member zone.
+
+When a catalog zone consumer of catalog zone `$NEWCATZ` receives an update of `$NEWCATZ` which adds or changes a member zone, *and* that consumer had the member zone associated with `$OLDCATZ`, *and* there is an `coo` property of the member zone in `$OLDCATZ` pointing to `$NEWCATS`, *only then* it will reconfigure the member zone with the for `$NEWCATZ` preconfigured settings.
```

Comment:
```suggestion
When a catalog zone consumer of catalog zone `$NEWCATZ` receives an update of `$NEWCATZ` which adds or changes a member zone, *and* that consumer had the member zone associated with `$OLDCATZ`, *and* there is an `coo` property of the member zone in `$OLDCATZ` pointing to `$NEWCATZ`, *only then* it will reconfigure the member zone with the for `$NEWCATZ` preconfigured settings.
```